javascript-if/main.js

```diff
@@ -0,0 +1,79 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+
+function isUnderFive(number) {
+  if (number < 5) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isEven(number) {
+  if (number % 2 === 0) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function startsWithJ(string) {
+  if (string[0] === 'J') {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrink(person) {
+  if (person.age >= 21) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrive(person) {
+  if (person.name === ' ' && person.age >= 16) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  if (person.age >= 16 && person.age >= 21) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function categorizeAcidity(pH) {
+  if (pH >= 15) {
+    return 'invalid pH level';
+  } else if (pH >= 8 && pH <= 14) {
+    return 'base';
+  } else if (pH === 7) {
+    return 'neutral';
+  } else if (pH <= 6) {
```

Comment:
We should also be checking for numbers less than 0. Things should only be considered acidic if they are less than 7 or greater than or equal to 0